src/cli.ts

```diff
@@ -229,4 +233,23 @@ program
     }
   });
 
+// Locations command
+program
+  .command('locations')
+  .description('List all locations to run the synthetics monitors')
+  .option('--url <url>', 'kibana URL to upload the monitors')
+  .requiredOption(
+    '--auth <auth>',
+    'API key used for Kibana authentication(https://www.elastic.co/guide/en/kibana/master/api-keys.html).'
+  )
+  .action(async (cmdOpts: LocationCmdOptions) => {
+    try {
+      console.log(await getLocations(cmdOpts));
+      // TODO: Display list of locations
```

Comment:
This will be covered as part of - https://github.com/elastic/synthetics/issues/551